packages/canary-react/src/component-tests/IcPaginationBar/IcPaginationBar.cy.tsx

```diff
@@ -0,0 +1,608 @@
+/* eslint-disable react/jsx-no-bind */
+/// <reference types="Cypress" />
+
+import React from "react";
+import { mount } from "cypress/react";
+import { IcPaginationBar } from "../../components";
+
+const PAGINATION_BAR = "ic-pagination-bar";
+const DEFAULT_THRESHOLD = 0.04;
+
+import { HAVE_LENGTH } from "../../../../react/src/component-tests/utils/constants";
```

Comment:
be good to use other constants in this file for consistency eg HAVE_TEXT, HAVE_CLASS, HAVE_ATTR